pkg/identity/cache.go

```diff
@@ -63,13 +65,44 @@ func GetIdentities() []*models.Identity {
 	return identities
 }
 
-func identityWatcher(owner IdentityAllocatorOwner) {
+func identityWatcher(owner IdentityAllocatorOwner, events allocator.AllocatorEventChan) {
+	var (
+		needPolicyUpdateMutex lock.RWMutex
```

Comment:
`s/RWMutex/Mutex` (RWMutex is slower if you don't do any reads. Yay more ops/s :tada: )